### ISSUE TITLE
Test suite

### ISSUE THREAD

**Issue by cjhowe-us** [choojs/create-choo-app#24]
Although `npm test` will run linters and dependency-check, it would be nice if we set up a test suite automatically. (maybe using mocha and zombie.js?) I think almost everyone will need this!

**Comment by yoshuawuyts** [comment]
Good point! – was thinking `tape` + `assert-html` should provide a good
baseline (:
